README.md

```diff
@@ -33,24 +33,38 @@ Braze's liquid is a subset of Shopify's liquid, hence some incompatible features
 #### Added
 
 * #### Filters
-    * [property_accessor][braze/property_accessor]: access value from hash
+
```

Comment:
Doesn't look like it's rendered correctly? you probably need to escape the `|`
![brazejs README md at 473b13c7e539830be638bd3992bc470db39c0292 · yq314 brazejs](https://user-images.githubusercontent.com/2336852/67636823-65d82300-f90f-11e9-980e-3f632f64b6f8.png)

Reply:
This is strange because the offending `|` character is within the backticks for the inline code block and shouldn't be behaving this way. It renders fine locally as well.

I tried pasting the same code in a markdown previewer and it renders it correctly too. 
https://jbt.github.io/markdown-editor/

let me try fixing the formatting again

Reply:
okay escaping it worked, seems like it's a github flavoured markdown thing